HPXMLtoOpenStudio/tests/test_defaults.rb

```diff
@@ -1178,21 +1178,37 @@ def test_central_air_conditioners
     hpxml_bldg.cooling_systems[0].crankcase_heater_watts = 40.0
     XMLHelper.write_file(hpxml.to_doc, @tmp_hpxml_path)
     _default_hpxml, default_hpxml_bldg = _test_measure()
-    _test_default_central_air_conditioner_values(default_hpxml_bldg.cooling_systems[0], 0.88, HPXML::HVACCompressorTypeVariableSpeed, 0.66, -0.11, -0.22, 12345, 12.0, 40.0, 1.0)
+    _test_default_central_air_conditioner_values(default_hpxml_bldg.cooling_systems[0], 0.88, HPXML::HVACCompressorTypeVariableSpeed, 0.66, nil, -0.11, -0.22, 12345, 12.0, 40.0, 1.0)
 
     # Test defaults - SEER2
     hpxml_bldg.cooling_systems[0].cooling_efficiency_seer = nil
     hpxml_bldg.cooling_systems[0].cooling_efficiency_seer2 = 11.4
     XMLHelper.write_file(hpxml.to_doc, @tmp_hpxml_path)
     _default_hpxml, default_hpxml_bldg = _test_measure()
-    _test_default_central_air_conditioner_values(default_hpxml_bldg.cooling_systems[0], 0.88, HPXML::HVACCompressorTypeVariableSpeed, 0.66, -0.11, -0.22, 12345, 12.0, 40.0, 1.0)
+    _test_default_central_air_conditioner_values(default_hpxml_bldg.cooling_systems[0], 0.88, HPXML::HVACCompressorTypeVariableSpeed, 0.66, nil, -0.11, -0.22, 12345, 12.0, 40.0, 1.0)
 
-    # Test autosizing with factors
+    # Test autosizing w/out limits
     hpxml_bldg.cooling_systems[0].cooling_capacity = nil
+    hpxml_bldg.cooling_systems[0].cooling_autosizing_limit = 30000
+    hpxml_bldg.cooling_systems[0].cooling_autosizing_factor = nil
+    XMLHelper.write_file(hpxml.to_doc, @tmp_hpxml_path)
+    _default_hpxml, default_hpxml_bldg = _test_measure()
+    _test_default_central_air_conditioner_values(default_hpxml_bldg.cooling_systems[0], 0.88, HPXML::HVACCompressorTypeVariableSpeed, 0.66, nil, -0.11, -0.22, nil, 12.0, 40.0, 1.0)
+    cooling_airflow_cfm = default_hpxml_bldg.cooling_systems[0].cooling_airflow_cfm
+    cooling_capacity = default_hpxml_bldg.cooling_systems[0].cooling_capacity
+
+    # Test autosizing with factors
```

Comment:
Hmm. I love that you added so much testing, but test_defaults.rb was really intended to check the defaulting logic for simple things, not the HVAC autosizing code. The autosizing factors are included here because they get defaulted to 1 when not provided. Likewise, it's fine to check that the autosizing limits default to nil when not provided if you want (even that's not really necessary).

But the code that checks whether the autosize factors are properly applied is [in test_hvac_sizing.rb](https://github.com/NREL/OpenStudio-HPXML/blob/b692d186bcd2e89656a318598ec8b569d05e98d4/HPXMLtoOpenStudio/tests/test_hvac_sizing.rb#L625). If it's not _too_ much effort, can you move these kinds of tests to there?